cmd/influxd/run/server_test.go

```diff
@@ -993,7 +993,7 @@ func TestServer_Query_Count(t *testing.T) {
 		&Query{
 			name:    "selecting count(*) should error",
 			command: `SELECT count(*) FROM db0.rp0.cpu`,
-			exp:     `{"results":[{"error":"expected field argument in count()"}]}`,
+			exp:     `{"error":"error parsing query: expected field argument in count()"}`,
```

Comment:
The error is now caught upstream further, so it isn't being wrapped by an error return again, or in this specific case, a results object.